support-frontend/assets/pages/supporter-plus-landing/twoStepPages/secondStepCheckout.tsx

```diff
@@ -50,10 +51,12 @@ export function SupporterPlusCheckout({
 }: {
 	thankYouRoute: string;
 }): JSX.Element {
+
 	const dispatch = useContributionsDispatch();
 	const { countryGroupId, countryId, currencyId } = useContributionsSelector(
 		(state) => state.common.internationalisation,
 	);
+	const country = countryGroups[countryGroupId].supportInternationalisationId;
```

Comment:
`country` feels a bit misleading here, how about -
`const { supportInternationalisationId } = countryGroups[countryGroupId];`

Reply:
Sure, happy to change the variable name. I'm just learning about deconstruction at the moment, so interested to know why you suggested have suggested it here instead of a simple name change only?

Reply:
Either is good - it's really a style thing, but I like that it's shorter -
`const { supportInternationalisationId } = countryGroups[countryGroupId];`
vs
`const supportInternationalisationId = countryGroups[countryGroupId].supportInternationalisationId;`